python/cuml/fil/fil.pyx

```diff
@@ -224,12 +239,14 @@ cdef class ForestInference_impl():
                                  TreeliteModel model,
                                  bool output_class,
                                  str algo,
-                                 float threshold):
+                                 float threshold,
+                                 str sparse):
```

Comment:
As with `sparse_t`, I think sparse sounds like a bool... would be good to do tree_type or something similar here too.

Reply:
Done.